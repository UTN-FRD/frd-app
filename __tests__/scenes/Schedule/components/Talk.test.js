// @flow
import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

import Talk from '../../../../app/scenes/Schedule/components/Talk';

describe('Schedule - Talk', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <Talk
          onPress={() => {}}
          speakerAvatarUri=""
          speakerName=""
          startTime=""
          status="present"
          title=""
        />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
