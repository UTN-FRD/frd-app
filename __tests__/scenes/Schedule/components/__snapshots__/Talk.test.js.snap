exports[`Schedule - Talk renders correctly 1`] = `
<View
  accessibilityComponentType={undefined}
  accessibilityLabel={undefined}
  accessibilityTraits={undefined}
  accessible={true}
  hasTVPreferredFocus={undefined}
  hitSlop={undefined}
  isTVSelectable={true}
  onLayout={undefined}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Array [
      Object {
        "backgroundColor": "transparent",
      },
      Object {
        "backgroundColor": "white",
      },
    ]
  }
  testID={undefined}
  tvParallaxProperties={undefined}>
  <View
    style={
      Array [
        Object {
          "alignItems": "stretch",
          "backgroundColor": "transparent",
          "flexDirection": "row",
        },
        undefined,
      ]
    }>
    <View
      style={
        Object {
          "backgroundColor": "#00A8D8",
          "width": 5,
        }
      }>
      <View
        style={
          Object {
            "transform": Array [
              Object {
                "translateX": 0,
              },
            ],
          }
        }>
        <Text
          accessible={true}
          allowFontScaling={false}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#00A8D8",
                "fontSize": 34,
              },
              Object {
                "backgroundColor": "transparent",
                "height": 34,
                "left": 0,
                "position": "absolute",
                "top": 10,
                "width": 34,
              },
              Object {
                "fontFamily": "Ionicons",
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
            ]
          }>
          
        </Text>
      </View>
    </View>
    <View
      style={
        Array [
          Object {
            "alignItems": "center",
            "backgroundColor": "transparent",
            "flexDirection": "row",
            "flexGrow": 1,
            "flexShrink": 1,
            "padding": 17,
          },
          undefined,
        ]
      }>
      <View
        style={
          Array [
            Object {
              "flexGrow": 1,
              "flexShrink": 1,
              "paddingRight": 12,
            },
            undefined,
          ]
        }>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "alignItems": "center",
                "flexDirection": "row",
                "marginBottom": 14,
              },
              Object {
                "color": "#666",
                "flexShrink": 1,
                "fontSize": 14,
                "fontWeight": "300",
              },
            ]
          }>
          
        </Text>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#383838",
                "fontSize": 17,
              },
              undefined,
            ]
          }>
          
        </Text>
      </View>
      <View
        style={
          Object {
            "alignItems": "center",
            "flexDirection": "row",
            "flexShrink": 0,
          }
        }>
        <View
          style={
            Array [
              Object {
                "backgroundColor": "#F4F4F4",
                "borderRadius": 44,
                "height": 44,
                "overflow": "hidden",
                "width": 44,
              },
              undefined,
            ]
          }>
          <Image
            source={
              Object {
                "uri": undefined,
              }
            }
            style={
              Object {
                "borderRadius": 0,
                "height": 44,
                "width": 44,
              }
            } />
        </View>
        <Text
          accessible={true}
          allowFontScaling={false}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#999",
                "fontSize": 20,
              },
              Object {
                "marginLeft": 17,
              },
              Object {
                "fontFamily": "Ionicons",
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
            ]
          }>
          
        </Text>
      </View>
    </View>
  </View>
</View>
`;
